### DISCUSSION TITLE
2024-08-26 release STS Error

### DISCUSSION THREAD

**Discussion by beeritis** [minio/minio#20341] [Q&A] [answered]
We have begun upgrading our cluster to 2024-08-26 , we were on  minio-2023111108 (out of date  I know!)

Had been reluctant to upgrade as the last time , it broke our LDAP authentication and stopped all our users from being able to login to the console. This was on 2024-07-04 release. 

Upgraded to the latest release (2024-08-26) , thankfully this error seems to now have been resolved. 

There is an error now everytime a user logs in even though they are authenticated successfully:

```
 API: SRPeerReplicateIAMItem
 Error: STS credential could not be verified: signature is invalid (*fmt.wrapError)
      4: internal/logger/logger.go:268:logger.LogIf()
      3: cmd/logging.go:46:cmd.adminLogIf()
      2: cmd/admin-handlers-site-replication.go:194:cmd.adminAPIHandlers.SRPeerReplicateIAMItem()
      1: net/http/server.go:2171:http.HandlerFunc.ServeHTTP()
```

It seems to be working okay but I wanted to check if this was a bug ? I.e can this safely be ignored as we plan to upgrade other hosts in our cluster to the latest version but want to make sure this doesn't cause problems.

## Expected Behavior
Log in successfully and no errors reported in the logs (as was before the upgrade)

## Current Behavior
We see the above error on every login.

## Possible Solution
Don'[t report the error / can STS be disabled as we don't use that when logging in ? ( To best of my knowledge)

## Steps to Reproduce (for bugs)
Upgraded to  2024-08-26 . Try and login via the MinIO console via LDAP , check the system logs via journalctl

## Your Environment

* Version used: 2024-08-26
* Server setup and configuration: RHEL 9.4 , standard system install (rpm packages) . Nginx in place as a reverse proxy

Thanks!

**Comment by harshavardhana** [comment, marked as answer]
It is not a bug. It is just a log to indicate that the decryption of an existing STS failed, but since STS are generally regenerated anyway, this is harmless.

**Reply by beeritis** [reply]
Thank you for confirming :)

**Comment by harshavardhana** [comment]
@vadmeste wasn't this supposed to be transparent?
